samples/python/chat_sample/README.md

```diff
@@ -22,3 +22,13 @@ To enable Unicode characters for Windows cmd open `Region` settings from `Contro
 Discrete GPUs (dGPUs) usually provide better performance compared to CPUs. It is recommended to run larger models on a dGPU with 32GB+ RAM. For example, the model meta-llama/Llama-2-13b-chat-hf can benefit from being run on a dGPU. Modify the source code to change the device for inference to the GPU.
 
 See https://github.com/openvinotoolkit/openvino.genai/blob/master/src/README.md#supported-models for the list of supported models.
+
+
+## Troubleshooting
+### Missing chat template
+
+If you encounter an exception indicating a missing "chat template" when launching the `ov::genai::LLMPipeline` in chat mode, it likely means the model was not tuned for chat functionality. To resolve this, manually add the chat template to tokenizer_config.json of your model.
```

Comment:
```suggestion
If you encounter an exception indicating a missing "chat template" when launching the `ov::genai::LLMPipeline` in chat mode, it likely means the model was not tuned for chat functionality. To work this around , manually add the chat template to tokenizer_config.json of your model.
```
It gives a perception that the suggested way resolves model tuning problem otherwise.

Reply:
done